dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/bnd.bnd

```diff
@@ -0,0 +1,29 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src,\
+    test-applications/EL50DefaultMethodBeanELResolverTest.war/src
+
+fat.project: true
+
+javac.source: 11
+javac.target: 11
+
+-buildpath: \
+    io.openliberty.jakarta.servlet.5.0;version=latest,\
```

Comment:
assume this will eventually change to io.openliberty.jakarta.servlet.6.0 when available?

Reply:
Correct

Reply:
@volosied can you open an issue for this one?

Reply:
Updated to use servlet 6.